src/build.sh

```diff
@@ -208,8 +218,9 @@ build_doc () {
 
 #===============================================================================
 ## main
-build_gerbil() {
+build_gerbil () {
```

Comment:
what's this space for?

Reply:
Oops, I was trying to bring some consistency in paren spacing, but it seems there are two competing styles in the same file.